pkg/apis/servicecatalog/validation/instance.go

```diff
@@ -60,6 +60,10 @@ func validateInstanceSpec(spec *sc.InstanceSpec, fldPath *field.Path, create boo
 		allErrs = append(allErrs, field.Invalid(fldPath.Child("planName"), spec.PlanName, msg))
 	}
 
+	if spec.Parameters != nil && spec.AlphaSecretParameters != nil {
+		allErrs = append(allErrs, field.Invalid(fldPath.Child("alphaSecretParameters"), spec.AlphaSecretParameters, "only one of 'Parameters' and 'AlphaSecretParameters' can be set at a time"))
```

Comment:
I seem to recall that there was concern in the past about secrets being limited in size so would forcing everything into the secret when perhaps just one field might need it be a concern?  Additionally, why don't we want to allow both?  Is this due to the conflict's name concern?